package.json

```diff
@@ -1666,6 +1682,12 @@
             "default": null,
             "description": "%configuration.dotnet.server.crashDumpPath%"
           },
+          "dotnet.server.enableXamlIntellisense": {
```

Comment:
How about enableXamlTools

Reply:
I think we ended up with `dotnet.enableXamlTools` and I'm adding a matching `dotnet.enableMobileHotReload`.